linux/guest_customization/check_gosc_support_status.yml

```diff
@@ -19,6 +19,16 @@
     skip_reason: "Blocked"
   when: enable_cloudinit_gosc and not vmtools_is_ovt
 
+- name: "Skip test case"
+  include_tasks: ../../common/skip_test_case.yml
+  vars:
+    skip_msg: "Test case '{{ ansible_play_name }}' is skipped because it's not supported"
+    skip_reason: "Not Supported"
+  when:
+    - guest_id == 'asianux8_64Guest'
+    - guest_os_ansible_distribution == 'MIRACLE'
```

Comment:
It doesn't need to check guest OS distribution, as no matter it is Miracle or not, as long as it is using 'asianux8_64Guest' in VM settings, it doesn't support GOSC on VC earlier than 8.0.0.

Reply:
Good comments. Thanks.

Reply:
@keirazhang 
Does it need to add the bellows condition ?   Thanks.
  - not enable_cloudinit_gosc